remote-access/ssh/passwordless.md

```diff
@@ -84,6 +84,25 @@ If this does not work, you can get assistance on the [Raspberry Pi forums](https
 
 **Note:** you can also send files over SSH using the `scp` command (secure copy). See the [SCP guide](scp.md) for more information.
 
+## Adjust permissions for your home and .ssh directories
+
+If you can't establish a connection after following the steps above there might be a problem with your directory permissions. First, you want to check the logs for any errors:
+
+```bash
+tail -f /var/log/secure
+# might return:
+Nov 23 12:31:26 raspberrypi sshd[9146]: Authentication refused: bad ownership or modes for directory /home/pi
+```
+
+If the logs says `Authentication refused: bad ownership or modes for directory /home/pi` there is a permission problem regarding your home directory. SSH needs your home and ~/.ssh directory forbid group write access. You can adjust the permissions using `chmod` (make sure to replace `user` with your username, e.g. `pi`):
+
```

Comment:
This sentence doesn't read right - where does forbid come in to it?

Reply:
I meant that those directories are not allowed to have write permission for groups. Sorry, I'm not a native english speaker and didn't know how to express it any better. Please feel free to correct the sentence and I'll add another commit (if that's the way PRs work).

Reply:
Here's what I would say:
```md
If the log says `Authentication refused: bad ownership or modes for directory /home/pi` there is a permission problem regarding your home directory. SSH needs your home and `~/.ssh` directory to not have group write access. You can adjust the permissions using `chmod` (make sure to replace `user` with your username, e.g. `pi`):
```